From 0f4cc881f4b350ad724e024f9a82ce4651ed2eda Mon Sep 17 00:00:00 2001
From: Zhang Rui <bbcallen@gmail.com>
Date: Mon, 27 Jun 2016 19:17:04 +0800
Subject: [PATCH 27/73] ff_ffplay: merge:
 8ef57a0d6154119e1a616dd8c29e8c32e35808a0
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

    Merge commit '41ed7ab45fc693f7d7fc35664c0233f4c32d69bb'

    * commit '41ed7ab45fc693f7d7fc35664c0233f4c32d69bb':
      cosmetics: Fix spelling mistakes

    Merged-by: Clément Bœsch <u@pkh.me>
---
 ijkmedia/ijkplayer/ff_ffplay.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ijkmedia/ijkplayer/ff_ffplay.c b/ijkmedia/ijkplayer/ff_ffplay.c
index 4ef8df6..88f62cc 100644
--- a/ijkmedia/ijkplayer/ff_ffplay.c
+++ b/ijkmedia/ijkplayer/ff_ffplay.c
@@ -2282,7 +2282,7 @@ static int stream_component_open(FFPlayer *ffp, int stream_index)
         /* init averaging filter */
         is->audio_diff_avg_coef  = exp(log(0.01) / AUDIO_DIFF_AVG_NB);
         is->audio_diff_avg_count = 0;
-        /* since we do not have a precise anough audio fifo fullness,
+        /* since we do not have a precise anough audio FIFO fullness,
            we correct audio sync only if larger than this threshold */
         is->audio_diff_threshold = 2.0 * is->audio_hw_buf_size / is->audio_tgt.bytes_per_sec;
 
-- 
1.9.1

